### ISSUE TITLE
Support for mandrill merge tag?

### ISSUE THREAD

**Issue by ouranos** [SparkPost/mandrill-sparkpost-templates#6]
Hi,

According to the README this tool should support MailChimp merge tags (`*|variable|*`).

However when I try to translate a really simple template:
```
Hi *|FIRST_NAME|*
```

It "translates" it as is. I'm expecting it to translate it to `Hi {{first_name}}`.

Is there something I'm missing?

**Comment by ewandennis** [comment]
Good point.  It looks like our merge tag support is a little too stringent in that it doesn't support underscores in tag names.  It also doesn't lowercase tag names during conversion and I think that might qualify as an unexpected effect.  It might be better as an option.

**Comment by ewandennis** [comment]
Resolved in v0.1.2.  Thanks for the feedback!

**Comment by ouranos** [comment]
Thanks!

It now works for variables but not for conditions.

For example:
```
*|IF:FIRST_NAME|*
Hi *|FIRST_NAME|*,
*|ELSE:|*
Hey there!
*|END:IF|*
```

**Comment by ewandennis** [comment]
Yup, we support basic merge tags syntax but not logic (yet) because the tool is broadly aimed at Mandrill handlebars syntax.  However, we are always open to pull requests and in fact the MailChimp merge tags support came from a contributor.

**Comment by ewandennis** [comment]
I'm going to rename this issue to "MailChimp Merge Tags" to keep the intent clear.

**Comment by ouranos** [comment]
Ok.
I'll see if I can find some time next week to try to implement this.

**Comment by ouranos** [comment]
See the attached pull request :)
